packages/runtime/src/next/tests/server.api-handler-manifest.test.ts

```diff
@@ -51,7 +51,7 @@ describe('MakeswiftApiHandler', () => {
         siteVersions: true,
         unstable_siteVersions: true,
         localizedPageSSR: true,
-        webhook: !pagesRouter,
+        webhook: true,
```

Comment:
`onPublish` event should be called in both Pages and App Router, see [this thread](https://makeswifthq.slack.com/archives/C014SCS4G75/p1753224257561999?thread_ts=1750462168.060599&cid=C014SCS4G75) for more details.